### PR TITLE
Documented the new Unique constraint

### PR DESCRIPTION
**Pull request by javiereguiluz**
Documents https://github.com/symfony/symfony/pull/26555

### CODE REVIEW

File: reference/constraints/Unique.rst [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            - `message`_
+            - `payload`_
+Class       :class:`Symfony\\Component\\Validator\\Constraints\\Unique`
+Validator   :class:`Symfony\\Component\\Validator\\Constraints\\UniqueValidator`
+==========  ===================================================================
+
+Basic Usage
+-----------
+
+This constraint can be applied to any property of type array or ``\Traversable``.
```
Comment (OskarStark) [inline reference/constraints/Unique.rst]:
```suggestion
This constraint can be applied to any property of type ``array`` or ``\Traversable``.
```

File: reference/constraints/Unique.rst [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    .. code-block:: php-annotations
+
+        // src/Entity/Person.php
+        namespace App\Entity;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class Person
+        {
+
```
Comment (OskarStark) [inline reference/constraints/Unique.rst]:
```suggestion

```

**Review by OskarStark — CHANGES_REQUESTED** [review]
Thank you Javier, great work again.

I left 2 minor comments

**Comment by OskarStark** [comment]
> Documents symfony/symfony#30706

I think you mixed some things up, and you meant: https://github.com/symfony/symfony/pull/26555

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Travis is failing with this:

```
/home/travis/build/symfony/symfony-docs/reference/constraints/Unique.rst:: 
WARNING: document isn't included in any toctree
```

But the new file is included in:

https://github.com/symfony/symfony-docs/pull/11247/files#diff-5c99f0007238ae4a681869aa8713c3fdR40

I'm lost :neutral_face:

**Comment by OskarStark** [comment]
I think you need to include it here

https://github.com/symfony/symfony-docs/blob/master/reference/constraints.rst

Your linked file is just an include but not an include in a toctree

**Comment by javiereguiluz** [comment]
You are right Oskar! Thanks a lot. I've fixed the missing reference.

**Review by OskarStark — APPROVED** [review]

**Pull request closed without merging**